components/dashboard/TextArea/index.tsx

```diff
@@ -49,14 +50,39 @@ const TextArea = ({ shiftRight, setShiftRight }: TextAreaProps) => {
       ).withConverter(notesConverter)
   );
 
-  const saveNoteChanges = async (noteId: string) => {
-    const lsInput = localStorage.getItem("editorContent");
-    const lsTitle = localStorage.getItem("editorTitle");
-    if (!isSynced && selectedNoteId && lsInput && lsTitle) {
+  const addToStore = async (
+    key: string,
+    value: { editorTitle: string; editorContent: string }
+  ) => {
+    set(key, value);
+  };
+
+  const getFromStore = async (
+    selectedNoteId: string
+  ): Promise<{ editorTitle: string; editorContent: string } | null> => {
```

Comment:
That causes TypeScript to give an error:

Type 'Promise<IDBObject> | null' is not a valid async function return type in ES5/ES3 because it does not refer to a Promise-compatible constructor value.

``Promise<IDBObject | null>`` Works

Reply:
Yes, that's fine. It should be <IDBObject | null>